go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,118 @@
+package ovn
+
+import (
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
```

Comment:
Would it make sense to have the definition of `retryObjs` and `retryObjEntry` here?